bpython/importcompletion.py

```diff
@@ -99,6 +100,19 @@ def module_attr_matches(name):
     return attr_matches(name, prefix="", only_modules=True)
 
 
+def try_to_import(module_name):
```

Comment:
Is this really necessary? Shouldn't those modules already be found by our normal module processing routines?